mwdb/model/migrations/versions/cdae4e840ceb_immutable_group.py

```diff
@@ -0,0 +1,30 @@
+"""immutable_group
+
+Revision ID: cdae4e840ceb
+Revises: 6db157d09a30
+Create Date: 2022-09-15 10:24:44.498960
+
+"""
+import sqlalchemy as sa
+from alembic import op
+
+# revision identifiers, used by Alembic.
+revision = "cdae4e840ceb"
+down_revision = "6db157d09a30"
+branch_labels = None
+depends_on = None
+
+
+def upgrade():
+    # ### commands auto generated by Alembic - please adjust! ###
+    op.add_column("group", sa.Column("immutable", sa.Boolean(), nullable=False))
```

Comment:
This line throws `column "immutable" of relation "group" contains null values`. We need to make nullable column first and set all values to True/False, then alter it to not-nullable.